ModelicaReference/package.mo

```diff
@@ -1902,20 +2050,18 @@ general Modelica class:
      actual arguments or default values, i.e., they may not be assigned
      values in the body of the function.</li>
 
-<li> A function may not be used in connections, may have no equations,
-     may have no initial algorithm, and can have at most one algorithm
-     section, which, if present, is the body of the function.</li>
+<li> A function may not be used in connections, may not have equations, may not have initial algorithms.</li>
```

Comment:
"may not" -> "cannot" or "must not"
Final ", may not" -> ", and cannot" or ", and must not"

Reply:
This is also from the 3.4 specification. Adding the last "and" seems like a typo-correction that we can do, but for "may not" it is a ibt complicated.

As I understand it "may not" is sometimes used similarly as "must not" (as in this case - obviously depending on the English variant), but it can mean other things as well so "must not" would be a clear improvement. 

However, there are about 51 uses of "may not" in the specification, and it seems likely that we should correct all of them in the actual specification and not one by one in this document.
(Currently in the specification a distinction is that "cannot" often/sometimes indicates that something is impossible, and "must not" and "may not" that it isn't legal; I don't know if that difference is proper English or not. But clearly one could write an equation-section in a function.)